gui/tools/autorepl.py

```diff
@@ -314,8 +314,9 @@ def write_results():
     sshproc = pipeopen('%s %s' % (sshcmd, rzfscmd))
     output, error = sshproc.communicate()
     remote_zfslist = {}
-    for i in re.sub(r'[ \t]+', ' ', output, flags=re.M).splitlines():
-        data = i.split()
+    for i in output.rstrip().split("\n"):
+        readonly_value = i.split()[-1]
+        data = [i.split("\t")[0].rstrip(), readonly_value]
```

Comment:
Why `rstrip`, dataset can have space as it's last name character e.g. `"data/work "`. 

What you need is
```
>>> "data/work \tyes".rsplit("\t",1)
['data/work ', 'yes']
```

Reply:
I used rstrip to remove the trailing newline that comes back from the remote ssh command.

Reply:
That is handled with the first rstrip in `output.rstrip()`

Reply:
Right, the 1st rstrip will strip the trailing newline but the second rstrip was to actually remove the trailing whitespace at the end of a dataset name. I did not realize that was even allowed or recognized in zfs....

Reply:
@themylogin okay, I've made the commit to account for trailing whitespace in the dataset name. Let me know if that works.